infrastructure/src/main/java/org/corfudb/infrastructure/datastore/KvDataStore.java

```diff
@@ -13,16 +13,7 @@
  *
  * <p>Created by mdhawan on 7/27/16.
  */
-public interface IDataStore {
-
-    @Deprecated
-    <T> void put(Class<T> tclass, String prefix, String key, T value);
-
-    @Deprecated
-    <T> T get(Class<T> tclass, String prefix, String key);
-
-    @Deprecated
-    <T> void delete(Class<T> tclass, String prefix, String key);
+public interface KvDataStore {
```

Comment:
Given the fact you are touching this class, can you please ass a class-level comment for this interface?

Reply:
It already has Javadoc